scripts/mockGuardianSpy.ts

```diff
@@ -0,0 +1,347 @@
+import {
+    Server,
+    ServerCredentials,
+} from '@grpc/grpc-js';
+import {MockGuardians} from "@certusone/wormhole-sdk/lib/cjs/mock";
+const GUARDIAN_PRIVATE_KEY = '0x5756c5018454b83225f8e8264651c3a8032c4ac931e1d310d81239754f1bb791';
```

Comment:
Don't like this const mixed up with the imports.

Reply:
Isn't actually needed, so removed.